src/views/BottomTabBar.js

```diff
@@ -23,6 +23,7 @@ export type TabBarOptions = {
   showIcon: boolean,
   labelStyle: any,
   tabStyle: any,
+  tabDirection: any,
```

Comment:
could use better type for this, unclear what it's meant to be and what the default is

Reply:
modeled the type after how I'd to it in typescript. Let me know if there's a better way to do this in flow. After this goes in I may take a stab at typing more of the `any`'s